server/models/userModel.js

```diff
@@ -26,6 +26,10 @@ const userSchemaModel = new mongoose.Schema({
     type: String,
     required: true,
   },
+  gender: {
+    type: String,
+    required: true,
```

Comment:
can you also add enum so the option are either male or female.
here is the link https://coursework.vschool.io/mongoose-schemas/